src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -40,19 +40,27 @@ For more detailed information about each of these, check out our [full Current L
 
 Once you’ve confirmed that your application only relies on [supported SDKs](#current-limitations), then the next step is to update them to the specified version or higher. Check out the migration documentation for the [SDKs](/platforms/) you’re using to learn everything you need to know to get up and running again with the latest Sentry features.
 
-## 2. Set a Uniform Sampling Rate
+## 2. Set a Uniform Sample Rate
 
-Next, go to **[Project] > Settings > Server-side Sampling** in [sentry.io](https://sentry.io) and set a uniform sampling rate for all transactions: this is the number of transactions that Sentry should accept from your application:
+Next, go to **[Project] > Settings > Server-Side Sampling** in [sentry.io](https://sentry.io) and set a sampling rate for all transactions: this is the number of transactions that Sentry should accept from your application:
 
-![Setting a uniform sampling rate for your project](set-uniform-sampling-rate.png)
+Before you set the rate, you'll need to tell us what your current SDK (or client-side) sample rate is:
+
+![Enter your client-side (SDK) sample rate](enter-sdk-sample-rate.png)
+
+With that done, set your uniform (global) sample rate:
+
+![Setting a sample rate for your project](set-global-sample-rate.png)
+
+Your client-side sample rate determines how many events are processed, or ingested, while the server-side sample rate determines how many events are then indexed, or retained. We use all your processed events to derive [performance metrics](/product/sentry-basics/metrics/), which makes us better able to provide you with statistically significant insights into the performance of your application. 
```

Comment:
Similar to my other comment, I'm not sure just providing the alternative name adds clarity. Maybe something like:
- processed (all those ingested by Sentry)
- indexed (all those retained by Sentry)